fair/identifiers.py

```diff
@@ -89,29 +87,15 @@ def check_ror(ror: str) -> typing.Dict:
         metadata from the given ID
     """
 
-    _url = f"{QUERY_URLS['ror']}{ror}"
-    _response = requests.get(_url)
-
-    _result_dict: typing.Dict[str, typing.Any] = {}
-
-    if _response.status_code != 200:
-        return _result_dict
-
-    if _response.json()["number_of_results"] == 0:
-        return _result_dict
-
-    _name = _response.json()["items"][0]["name"]
-    _result_dict["name"] = _name
-    _result_dict["family_name"] = _name
-    _result_dict["given_names"] = None
-    _result_dict["ror"] = ror
-    _result_dict["uri"] = f'{ID_URIS["ror"]}{ror}'
+    _result_dict = _check_generic_ror(ror)
+    if _result_dict:
+        _result_dict["ror"] = ror
 
     return _result_dict
 
 
 def check_grid(grid_id: str) -> typing.Dict:
-    """Checks if valid GRID ID using GRID public api
+    """Checks if valid GRID ID using ROR public api
```

Comment:
```suggestion
    """Checks if valid GRID ID using ROR (https://ror.org/) public api
```

Reply:
It seems GRID ID is no longer a thing and it's now ROR ID. I guess this would need a schema change to fully implement this and that's probably not in scope here, but the docstring and maybe variable names could reflect this a bit more?

Reply:
I agree on the docstring suggestion, at the moment the GRID ID is usable by ROR as a parameter.